host_applications/linux/apps/raspicam/CMakeLists.txt

```diff
@@ -54,9 +54,9 @@ add_executable(raspividyuv  ${COMMON_SOURCES} RaspiVidYUV.c)
 
 set (MMAL_LIBS mmal_core mmal_util mmal_vc_client)
 
-target_link_libraries(raspistill ${MMAL_LIBS} vcos bcm_host brcmGLESv2 brcmEGL m dl)
-target_link_libraries(raspiyuv   ${MMAL_LIBS} vcos bcm_host)
-target_link_libraries(raspivid   ${MMAL_LIBS} vcos bcm_host)
-target_link_libraries(raspividyuv   ${MMAL_LIBS} vcos bcm_host)
+target_link_libraries(raspistill ${MMAL_LIBS} vcos bcm_host brcmGLESv2 brcmEGL m dl wiringPi)
+target_link_libraries(raspiyuv   ${MMAL_LIBS} vcos bcm_host wiringPi)
+target_link_libraries(raspivid   ${MMAL_LIBS} vcos bcm_host wiringPi)
+target_link_libraries(raspividyuv   ${MMAL_LIBS} vcos bcm_host wiringPi)
```

Comment:
I wonder if this means the LED will turn on when we capture a video... did you check what happens in `raspistill -k`?

Reply:
good question

I haven't dug into how this is installed, but could our custom version be built with a different binary name that sits alongside raspistill (`raspistill_led_on` or something)?

Reply:
probably.. but it feels like that might be past the 80/20 point. given our experience just trying to understand the build script well enough to figure out where to insert the library, it seems like it would probably take a while to make it work.

Reply:
oh, and I just tested, and raspistill -k does *not* turn on the LED

Reply:
also, raspivid and raspiyuv don't turn on the led either